figgy-fixture-container/build-and-push.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+./create-fixture-exports.sh
+echo "Enter lastpass username:" 
+read USERNAME
+lpass login $USERNAME
+export CR_PAT=$(lpass show "Shared-ITIMS-Passwords/dpul_collections_fixture_container_github_token" --notes)
+echo $CR_PAT | docker login ghcr.io -u pulbot --password-stdin
```

Comment:
What does CR_PAT mean?

Reply:
No clue, I copied this from a different script we have in figgy. 🤷